config/host/test.exs

```diff
@@ -1,6 +1,19 @@
 import Config
 
-config :xebow, settings_path: "priv/settings_test"
+config :xebow,
+  settings_path: "priv/settings_test",
+  layout: [
```

Comment:
We might want to think about a layout fixture for the test suite rather than hard-coding an implementation in the config. It may be harder to change the layout for specific tests this way.

I think the method implemented here would be better for integration testing.